docs/user-guides/configuration-guide.md

```diff
@@ -742,6 +743,11 @@ The following table describes the subfields for the `streaming` field:
     By default, the toolkit streams the chunks as soon as possible and before applying output rails to them.
 
   - `True`
+
+* - streaming.enabled
+  - When set to True enable the execution of the output rails in streaming mode.
```

Comment:
Do you mind putting this row in alphabetical order?

Thanks for following the "When set..." pattern.  Do you mind enclosing True in backticks (`True`) and following with a comma?  Maybe also revise to "When set to `True`, the toolkit executes output rails in streaming mode."